src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -308,63 +308,63 @@
             </trans-unit>
             <trans-unit id="80">
                 <source>This value does not match the expected {{ charset }} charset.</source>
-                <target>Detta värde har inte den förväntade teckenkodningen {{ charset }}.</target>
+                <target>Värdet har inte den förväntade teckenkodningen {{ charset }}.</target>
             </trans-unit>
             <trans-unit id="81" resname="This is not a valid Business Identifier Code (BIC).">
                 <source>This value is not a valid Business Identifier Code (BIC).</source>
-                <target state="needs-review-translation">Detta värde är inte en giltig Företagsidentifieringskod (BIC).</target>
+                <target>Värdet är inte en giltig Företagsidentifieringskod (BIC).</target>
```

Comment:
Im hesitant of this. 

If you asked me for my "Företagsidentifieringskod", I would assume you asked for my business organization number or possibly my LEI. 

I think a better translation would be: 

```suggestion
                <target>Värdet är inte en giltig BIC-kod.</target>
```

My bank (Swedbank) refers to this as just "BIC".